.github/workflows/release.yml

```diff
@@ -53,12 +53,13 @@ jobs:
           PROJECTS=${{ inputs.projects }}
 
           if [[ $PROJECTS == "all" ]]; then
+            yarn nx affected --target=version
```

Comment:
```suggestion
            echo $PROJECTS
            yarn nx affected --target=version --base=last-release
```

Reply:
Not sure about this, but have you tested this??

Affected returns everything including apps and libs... Running this might also try to run `version` target in apps as well which will fail coz there is no `version` target in apps.

Reply:
hi, @pavandv! how you doing there?

sorry for the delay.

i tested it, but it didn't worked as expected.

turns out the affected will trigger `version` for some libs that didn't change the code at all.

e.g.:

- `nextjs-mf` changes will trigger changes for `nextjs-mf` only, because isn't a dependency
- `node` will trigger changes for `nextjs-mf` and `node`, because `nextjs-mf` is a dependency
- `typescript`will trigger changes for `typescript` only
- `utils` will trigger changes for every lib